sky/cli.py

```diff
@@ -1325,6 +1325,7 @@ def launch(
     In both cases, the commands are run under the task's workdir (if specified)
     and they undergo job queue scheduling.
     """
+    backend_utils.check_network_connection()
```

Comment:
We should wrap this in a `with ux_utils.print_exception_no_traceback():` to avoid printing a long stack trace

Reply:
Did end up wrapping it in this but stack trace printed is the long list of exceptions still